dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ExclusionFileParser.java

```diff
@@ -62,11 +68,25 @@
   static ImmutableList<LinkageErrorMatcher> parse(Path exclusionFile)
       throws SAXException, IOException {
 
+    // DraconianErrorHandler throws SAXException upon invalid structure
+    ValidationDriver validationDriver =
```

Comment:
I'm really not convinced that we want to do this when processing the documents.

Reply:
Memo:

Cost
- performance for high qps services. (not this case)
- too restrictive. For example, users cannot put their exclusion files in pom.xml.
- adding dependencies. (fine)
- adding validation makes a system more fragile